pkgs/tools/misc/cp-p/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "cp-p";
+  pname = "cp-p";
```

Comment:
```suggestion
  pname = "cp-p";
  version = "unstable-2022-08-07";
```

Reply:
When I wanted to build that derivation without `name` it didn't want to build

Reply:
name is just pname+version, please make the requested change